experience.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en" style="background-color:rgb(48, 48, 48); color: rgb(226, 225, 223);">
+<head>
```

Comment:
Noticed some inconsistencies here - the head tag should only contain metadata about the page, ie information that will affect the page's display, but information that we won't see directly - this is why the link to the css page should go here.  Any element - nav, header, etc should be in the body elements, as these are elements we'd be wanting to view in the document itself.